Tools/Pipeline/Common/AssemblyAttributes.cs

```diff
@@ -81,6 +81,14 @@ public static string AssemblyCompany
 				return ((AssemblyCompanyAttribute)attributes[0]).Company;
 			}
 		}
+
+        public static string[] Developers
+        {
+            get
+            {
+                return new string[] { "cra0zy (Harry)", "dellis1972 (Dean Ellis)", "dneelyep (Daniel Neel)", "emarcotte (Eugene Marcotte)", "JamesLupiani (James Lupiani)", "jamesford42 (James Ford)", "KonajuGames (Steve 'Sly' Williams)", "mgarstenauer (Martin Garstenauer)", "nightst4r (Mark)", "nkast (Nikos Kastellanos)", "polsys (Petri Laarne)", "RayBatts (Ray Batts)", "Shqrdx (Yuri Roubinsky)", "thiagoabreu (Thiago 'Jedi' Abreu)", "tomspilman (Tom Spilman)" };
```

Comment:
Agreed. No other tool in MonoGame lists individual contributors. It should
just mention MonoGame and direct them to the website for more information.
If someone really wants to find out who worked on it, they can go to source
control history.